### PR TITLE
Added a few features

### PR DESCRIPTION
**Pull request by yunquliu**
1. Print the time stamp in human read mode instead of unix mode
2. Parse the ScriptSig into script and InputPubkey
3. Parse the ScriptPubkey into OP_CODE and pubkeyHash
4. Added a output.txt which is a one block sample

To do so, added in a new opcode.y file

	modified:   block.py
	new file:   opcode.py
	new file:   output.txt
	modified:   sight.py

### CODE REVIEW

**Comment by AlexChambers** [comment]
@yunquliu 
I'm wondering if @tenthirtyone has been to busy to review your PR request or if it needs to be on a named feature branch (i.e. `git push origin my-new-feature`)

**Comment by tenthirtyone** [comment]
Little busy working on https://igave.io

I'll give this a look

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+This is a fork of block chain tools.
```
Comment (tenthirtyone) [inline README.md:1]: Can you credit your contribution here instead?

**Review by tenthirtyone — CHANGES_REQUESTED** [review]
Nice work. It always bothered me I never got to the Script. Thank you

**Comment by yunquliu** [comment]
My Pleasure.

 

Leon

 

发件人: Alex [mailto:notifications@github.com] 
发送时间: 2018年2月2日 13:55
收件人: tenthirtyone/blocktools <blocktools@noreply.github.com>
抄送: yunquliu <yunqu.liu@gmail.com>; Mention <mention@noreply.github.com>
主题: Re: [tenthirtyone/blocktools] Added a few features (#8)

 

@tenthirtyone requested changes on this pull request.

Nice work. It always bothered me I never got to the Script. Thank you

  _____  

In README.md <https://github.com/tenthirtyone/blocktools/pull/8#discussion_r165563531> :

> @@ -1,3 +1,9 @@
+This is a fork of block chain tools.

Can you credit your contribution here instead?

**Pull request merged**